compiler/rustc_expand/src/build.rs

```diff
@@ -144,7 +144,14 @@ impl<'a> ExtCtxt<'a> {
         ast::Stmt { id: ast::DUMMY_NODE_ID, span: expr.span, kind: ast::StmtKind::Expr(expr) }
     }
 
-    pub fn stmt_let(&self, sp: Span, mutbl: bool, ident: Ident, ex: P<ast::Expr>) -> ast::Stmt {
+    pub fn stmt_let(
+        &self,
+        attrs: Vec<ast::Attribute>,
```

Comment:
```suggestion
        attrs: AttrVec,
```
If we need an `AttrVec`, let's pass an `AttrVec` as well.

(It can be constructed as `Default::default()` on the call sites to avoid importing `AttrVec`.)